src/stories/SegmentedPicker/index.tsx

```diff
@@ -0,0 +1,67 @@
+import React, { useState } from "react";
+import clsx from "clsx";
+import { SegmentedPickerProps } from "./type";
+
+export const SegmentedPicker: React.FC<SegmentedPickerProps> = ({
+  className,
+  segments,
+  onSelect,
+}) => {
+  const [selectedSegment, setSelectedSegment] = useState(segments[0]);
```

Comment:
please provide a props value for have a default selected

Reply:
change:
  const [selectedSegment, setSelectedSegment] = useState<Segment<any> | null>(
    segments[0].value !== undefined ? segments[0].value : null
  );